README.md

```diff
@@ -1,33 +1,27 @@
-# Spark Integration Hub K8s Operator
+# Integration Hub for Apache Spark K8s Operator
 
-The Spark Configuration Hub charm, is the charm that enables the integration of 
-the Charmed Spark solution with existing charms (PostgreSQL, S3-integrator, etc) 
-and bundles (COS, Identity, etc) and properly configure the corresponding 
-option in the Spark ecosystem.
+The Integration Hub for Apache Spark K8s charm is the charm that enables the integration of the Charmed Apache Spark solution with existing charms (PostgreSQL, S3-integrator, etc) and bundles (COS, Identity, etc) and properly configure the corresponding option in the Apache Spark ecosystem.
 
-The main tasks of the configuration hub charm are the following:
+The main tasks of the Integration Hub charm are the following:
 
-* Handle relations with other charms and translate them in configuration options 
+- Handle relations with other charms and translate them in configuration options
   that Spark can handle (like the Spark History Server charm does with the S3-integrator).
-* Store the configuration options in Kubernetes secrets for the service account
-  managed by the Charmed Spark solution (all service account are labeled with predefined labels)
-* Handle the addition/removal of service accounts with corresponding updates of 
+- Store the configuration options in Kubernetes secrets for the service account
+  managed by the Charmed Apache Spark solution (all service account are labeled with predefined labels)
+- Handle the addition/removal of service accounts with corresponding updates of
   configurations.
-* Handle updates from the related charms (i.e., new database credentials, 
+- Handle updates from the related charms (i.e., new database credentials,
   change of endpoints, updated S3 credentials.)
-* The configuration hub charm may also be able to create and configure services
+- The Integration Hub charm may also be able to create and configure services
   accounts that are able to run Spark jobs. This can be a possible scenario if
-  we relate our solution with Kubeflow. 
+  we relate our solution with Kubeflow.
```

Comment:
Kubeflow -> Kyuubi 
For the time being there is no official integration between Charmed Spark and Kubeflow

Reply:
@deusebio  What do you think ?

Reply:
Remove this sentence in 08621630f3a9c5691571727557706c4d67a274d9. It was not that important IMO, and I would rather avoid any kind of conditional in the readme